general/payment_acquirers/stripe.rst

```diff
@@ -0,0 +1,27 @@
+======
+Stripe
+======
+
+`Stripe <https://stripe.com/>`__ is a United States-based online payment solution
+provider, allowing businesses to accept **credit cards** and other payment methods.
+
+Local Payment Methods
+=====================
+
+`Local payment methods <https://stripe.com/payments/payment-methods-guide>`__ are payment methods which are
+only available for certain merchants and customers countries and currencies.
+
+Odoo supports the integration of the following local payment methods:
+
+- Bancontact
+- EPS
+- Giropay
+- iDeal:
+- Przelewy24 (P24)
+
+In order to support one local payment method, you must add its icon to the supported payment method icons:
+Payment Acquirers > Stripe > Configuration > Supported Payment Icons.
+
+If the icon of a payment method doesn't exist at all, **Odoo will offer the corresponding payment method by default**.
+In order to disable such a payment method, update the Stripe module to get the new icons or create the icon yourself and
+don't add it to the supported payment icons.
```

Comment:
```suggestion
.. note::
   If a payment icon does not exist at all in the database, the corresponding local payment method
   is always offered to customers.
```
I wouldn't recommend users to create the icon by themselves since they must figure out the exact name to use, and I would certainly not recommend them to upgrade their `payment` module just for this.
Instead, let's just stick to the description of the behavior and we'll see later is something more is required.

Reply:
How will the customers get the icon I added again ? I thought they had to update payment_stripe to get them. Well, I guess the information will be part of the support mail we'll send, so indeed it doens't need to be here.

Reply:
The icons are defined in the `payment` module, not in `payment_stripe`. Upgrading such a module is not a trivial action so we shouldn't rely on that to ensure a working fix.